src/freenet/clients/http/staticfiles/js/firsttimewizard.js

```diff
@@ -0,0 +1,126 @@
+var slideToggler, trigger,
+    bind = function(fn, me) {
+        return function() {
+        return fn.apply(me, arguments);
+    };
+    },
+    indexOf = [].indexOf || function(item) {
+        for (var i = 0, l = this.length; i < l; i++) {
+        if (i in this && this[i] === item) return i;
+    }
+
+    return -1;
+};
+
+slideToggler = (function() {
+    function slideToggler(el1) {
+        if (el1) {
+            this.el = el1;
+            this.toggle = bind(this.toggle, this);
+            this.height = this.getHeight();
+        } else {
+            return;
+        }
+    }
+
+    slideToggler.prototype.getHeight = function() {
+        var clone;
+        if (this.el.clientHeight > 10) {
+            return this.el.clientHeight;
+        }
+        clone = this.el.cloneNode(true);
+        clone.style.cssText = 'position: absolute; visibility: hidden; display: block;';
+        this.el.parentNode.appendChild(clone);
+        this.height = clone.clientHeight;
+        this.el.parentNode.removeChild(clone);
+        return this.height;
+    };
+
+    slideToggler.prototype.toggle = function(time) {
+        var currHeight, disp, el, end, init, ref, repeat, start;
+        el = this.el;
+        if (!el) {
+            return;
+        }
+        if (!(this.height > 0)) {
+            this.height = this.getHeight();
+        }
+        if (time == null) {
+            time = this.height;
+        }
+        currHeight = el.clientHeight * (getComputedStyle(el).display !== 'none');
+        ref = currHeight > this.height / 2 ? [this.height, 0] : [0, this.height];
+        start = ref[0];
+        end = ref[1];
+        disp = end - start;
+        this.el.classList[end === 0 ? 'remove' : 'add']('open');
+        this.el.style.cssText = "overflow: hidden; display: block; padding-top: 0; padding-bottom: 0";
+        init = (new Date).getTime();
+        repeat = function() {
+        var instance, ref1, repeatLoop, results, step;
```

Comment:
here the indentation seems to be different from the logical code — the function body should be more indented.